core/lib/server/pub.ts

```diff
@@ -395,19 +395,17 @@ export const createPubRecursiveNew = async <Body extends CreatePubRequestBodyWit
 	const parentId = parent?.id ?? body.parentId;
 	const stageId = body.stageId;
 
-	const pubFieldsForPubTypeObject = await getPubFields({
```

Comment:
We don't necessarily want to restrict our pub values to only those of the pub type, since our forms might add different values. So now we query for pub fields by community instead (which is likely a cached query)